Zilon.Core/Zilon.Core.Benckmarks.Fow/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using BenchmarkDotNet.Running;
+
+namespace Zilon.Core.Benckmarks.Fow
+{
+    class Program
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5305216)